docs/release-process.md

```diff
@@ -2,6 +2,18 @@
 
 ## Web wallet
 
+Make a release:
+
+- ```sh
```

Comment:
id this shell cmd needed in a flow ?

Reply:
It's different from the official towncrier, so i think its good to point it out whereever its used. Would be better if changelog.js checked if correct towncrier exists and printed "pip install", but that affects multiple repos